stable/prometheus-postgres-exporter/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Should be conditionally created based on `serviceAccount.create`

Reply:
done

Reply:
done